content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+channel needs.
+
+### Attention seeking
+
+If the bad behaviour is someone wanting attention, responding to the
+attention-seeking proves that it works, and they'll keep doing it. If it
+doesn't work, they'll get bored and probably move on.
+
+### Bad language and frustration
+
+If someone is a bit snappy or using bad words out of frustration, scolding
+them can make them more frustrated.
+
+In a support channel, helping a user with their problem is much more important
+than trying to make them ask in the "right" way. So unless it becomes a habit
+or makes the channel uncomfortable, it's not worth derailing the support
+process.
+
+### Typos and non-fluent language
+
+Don't draw attention to someone else's typos or pick on them for their use of
+abbreviations like 'u', unless they specifically request help with wording.
+They could be typing from a phone keyboard, be non-fluent in the language, or
+have some condition that affects their typing.
+
+They may be sensitive and feel unwelcome if people focus more on their wording
+and ignore their questions. If you can still understand what someone is
+asking, answer it and ignore the typos.
+
+### Crossposting
+
+It may feel like a waste to answer someone asking the same question in
+multiple channels. Sometimes people don't have much time, or maybe they have
+a boss hurrying them! Seeking second opinions is also a perfectly normal
+social practice; people do it all the time.
+
+Confronting people about occasional crossposting makes the channel feel
+hostile, and people feel like they are under surveillance, regardless of why
+they do it. If someone wants to answer the question, let them. If you don't
+want to answer, ignore it quietly.
+
+## They are being rude
+
+While it would be very nice if people were never rude, it **is** going to
+happen. However,
+[don't automatically assume people are rude](catalyst#assume-good-faith).
+They may have different cultural norms or a language barrier that
+unintentionally changes their perceived tone.
+
+It is best to let small things slide and not raise the channel temperature by
+calling them out. If a person's demeanor bothers you, quietly refrain from
+engaging with them instead of telling them off. If a channel member is
+hostile, it is important not to worsen the situation by responding in kind.
+
+## They are in the wrong place
+
+While you may know how to navigate IRC, other people might be confused or not
+know where to begin. Instead of scolding people for getting the wrong channel,
+it is better to say you do not know the answer and tell them the correct
+channel to ask.
+
+When you redirect people, first check if they can join the destination
+channel. Otherwise, it might seem like you're pushing them away. You may need
+to `/whois` them to find out if they are identified or not:
+
+```irc
+/whois Nickname
+```
+
+If they are identified, there will be a line saying so. It looks like this:
+
+```irc
+[Nickname] is logged in as Accountname
```

Comment:
```suggestion
<nickname> is logged in as Accountname
```

I'd use `<>` to signify that it's "required"/always present, maybe an overfluffy interpretation of the styleguide

Reply:
this is a literal quote though